src/net/gateway.ts

```diff
@@ -23,6 +22,9 @@ export type IGateway = InstanceType<typeof Gateway>
 export class Gateway {
     protected dvote: DVoteGateway = null
     protected web3: Web3Gateway = null
+    protected archiveIpnsId: string
+    protected environment: VocdoniEnvironment
```

Comment:
Why do we need this? UI projects will have an environment variable that will know this in advance.